### ISSUE TITLE
Remove resty

### ISSUE THREAD

**Issue by toolbear** [Homebrew/homebrew#39978]
https://github.com/Homebrew/homebrew/blob/master/Library/Formula/resty.rb
https://github.com/micha/resty

* `resty` is meant to be sourced ala `source resty`; this adds a `resty` function in your shell. This formula installs an executable that makes `resty` a no-op unless you do `source /usr/local/bin/resty`

* formula installs `pp` and `pypp`, tiny shims around Perl and Python JSON pretty printers; presumably better left for a separate, dedicated formulae

* `/usr/bin/pp` is the "PAR Packager", installing this formula shadows that

Alternatives to full removal:

* [ ] see if upstream is receptive to a patch to make `resty` suggest `source resty` when invoked as an executable
* [ ] add post-install instructions to add `source /usr/local/bin/resty` to your init scripts
  * [ ] bonus for patching upstream to allow `eval $(resty --init)`
* [ ] only install `resty`, don't install `pp` and `pypp`

**Comment by toolbear** [comment]
TL;DR - `brew install resty` exposes a `resty` executable in my path that does nothing

**Comment by mikemcquaid** [comment]
We have people who use this so I think removing it is overkill. Moving the `resty` script from `bin` to `share` seems like a sensible move.

**Comment by DomT4** [comment]
Any chance of a PR here? Perhaps given the `/usr/bin/pp` shadowing we could dump everything in `libexec` and then selectively symlink, or symlink with the utilities prefixed with the formula name?

We have various caveats in place for scripts that need sourcing, so that could be suitable here as well if they aren't in place already.

**Comment by DomT4** [comment]
Any update on this?

**Comment by bfontaine** [comment]
Closing due to no response in two weeks, feel free to make a PR as suggested by @DomT4. Thanks!
